serde/src/lib.rs

```diff
@@ -10,36 +10,64 @@ pub use serde_derive::{Deserialize, Serialize};
 
 /// Deserialization Error. This error is a wrapper over `postcard::Error`.
```

Comment:
The error is strictly speaking no longer a wrapper over `postcard::Error`

Reply:
Fixed.